src/app/ddDrakeModel.h

```diff
@@ -78,6 +84,10 @@ class DD_APP_EXPORT ddDrakeModel : public QObject
   static void addPackageSearchPath(const QString& searchPath);
   static QString findPackageDirectory(const QString& packageName);
 
+  #ifdef DRAKE_OH_FORK
+  void doKinematics(const QVector<double>& q, const QVector<double>& v, bool compute_gradients = false, bool compute_JdotV = true);
```

Comment:
I'm not using this currently. I think it would be nice to be able to call doKinematics with positions and velocities. Currently you can only do it with positions, through the setJointPositions method, this implicitly sets all velocities to zero. I'm not sure worried about supporting this use case however since really people should directly use the drake python api instead of going through director.